roles/provision-unifiedpush-apb/defaults/main.yml

```diff
@@ -1,14 +1,17 @@
 # UPS Values
 ups_secret_name: "unified-push-server"
-ups_image: " docker.io/aerogear/unifiedpush-wildfly"
-ups_image_tag: "1.3.0-no-auth.Final"
+ups_image: " docker.io/matzew/unifiedpush-psql"
+ups_image_tag: "latest"
```

Comment:
yes, for now (see coming PRs) this is the "test" image to use .. and to avoid confusion :-)